newsfragments/36922.feature.rst

```diff
@@ -0,0 +1 @@
+Support for configmaps in the template fields in the KubernetesPodOperator
```

Comment:
Please remove this file.
This change is in providers we dont need newsfragment for providers :)